raster/src/main/scala/geotrellis/raster/io/geotiff/GeoTiff.scala

```diff
@@ -52,6 +52,19 @@ trait GeoTiff[T <: CellGrid] extends GeoTiffData {
  * Companion object to GeoTiff
  */
 object GeoTiff {
+  def readMultiBand(path: String): MultibandGeoTiff =
+    MultibandGeoTiff(path)
+
+  def readSingleBand(path: String): SinglebandGeoTiff =
```

Comment:
should be `readSingleband` (lower case b) to be consistent with methods on the readers